.yarn/patches/@metamask-assets-controllers-npm-30.0.0-8747c20871.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/dist/chunk-Z6TBQQE5.js b/dist/chunk-Z6TBQQE5.js
```

Comment:
Let's not forget to follow-up after release of https://github.com/MetaMackask/metamask-extension/pull/24913 so that patching won't be needed at all for this package anymore :rocket: